test/manifest-date.js

```diff
@@ -4,23 +4,28 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-const tap = require("tap").test;
-const {
-  generateCatalogueManifest,
-  generateVectorManifest,
-} = require("../scripts/generate-manifest");
+import fs from "node:fs";
 
-const { getExpectedVector } = require("./manifest-v7.14");
+import { test as tap } from "tap";
 
-const {
+import {
+  generateCatalogueManifest,
+  generateVectorManifest,
+} from "../scripts/generate-manifest.js";
+import { getExpectedVector } from "./manifest-v7.14.js";
+import {
   formatDateToIso,
   checkDateVersion,
   coerceToDateSemver,
-} = require("../scripts/date-versions");
-
-const sources = require("./fixtures/valid-sources/sources.json");
-const fieldInfo = require("./fixtures/fieldInfo.json");
-const constants = require("../scripts/constants");
+} from "../scripts/date-versions.js";
+import constants from "../scripts/constants.js";
+
+const sources = JSON.parse(
```

Comment:
I did not find a better way to import these JSON files. Is not super clean but :shrug:

Reply:
AFAIK, that's the best way to import JSON.